syntax = "proto3";

package demo;

service Demo {
    rpc GetModelWithId(ModelId) returns (Model) {}
}

message ModelId {
    int32 id = 1;
}

message Model {
    string text_value = 1;
    string select_value = 2;
}
